test/sample.cc

```diff
@@ -48,7 +48,7 @@ TEST(Sample, create_tile) {
   // input for gzip
   auto src_func = [&tile](z_stream& s) -> int {
     s.next_in = static_cast<Byte*>(static_cast<void*>(tile.data()));
-    s.avail_in = static_cast<unsigned int>(tile.size() * sizeof(decltype(tile)::value_type));
+    s.avail_in = static_cast<unsigned int>(tile.size() * sizeof(int16_t));
```

Comment:
MSVC wasn't happy about the `decltype` here. Elsewhere in the file this simply used `sizeof(int16_t)` so I just defaulted to that to fix the build.